packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -31,22 +39,34 @@ const mapToLPADashboardDisplayData = (appealCaseData) => ({
 	decision: appealCaseData.outcome
 });
 
+/**
+ * @param {AppealCaseWithAppellant} appealCaseData
+ */
+
```

Comment:
Can we remove all the gaps between jsdoc and function please?

Reply:
done